orttraining/orttraining/python/training/ortmodule.py

```diff
@@ -306,24 +323,28 @@ def backward(ctx, *grad_outputs):
                 raise NotImplementedError(
                     "The model's forward method has **kwargs parameter which is currently not supported.")
 
-        self._onnx_inference = None
+        self._exported_onnx_training_model = None
+        self._exported_onnx_inference_model = None
 
         # Related to training graph shape inference
         self._current_input_shape = None
         # default execution order is priority-based for both dynamic/static shape input for now
         # if we observe benefit of static shape, we can expose this flag to user
         self._use_static_shape = False
-        self._module_gradient_graph_builder = None
+        self._module_gradient_graph_builder_training = None
+        self._module_gradient_graph_builder_inference = None
```

Comment:
```suggestion
        self._module_gradient_graph_builder = None
        self._module_inference_graph_builder = None
```
It sounds odd having a `self._module_gradient_graph_builder_inference` as inference doesnt need gradients and `self._module_gradient_graph_builder_training` is slightly redundant as a gradient builder is implicitly for a training graph

Reply:
I definitely agree. It is the name of the class that does the graph optimizations for both the inference graph as well as the training graph which is used here. I can change the name of the class to ```GraphBuilder``` instead of ```ModuleGrdientGraphBuilder``` (and change the name of the variable) or just change the name of this variable here.

Reply:
Renamed ```module_gradient_graph_builder``` to ```module_grapher_builder```.